set-mongo/src/main/java/eu/europeana/set/mongo/model/internal/PersistentUserSet.java

```diff
@@ -20,6 +20,7 @@
 		public static final String FIELD_TITLE = "title";
 		public static final String FIELD_DESCRIPTION = "description";
 		public static final String FIELD_SET_TYPE = "setType";
+		public static final String FIELD_TYPE = "type";
```

Comment:
seems to remove the FIELD_SET_TYPE constant, please remove the unused constant

Reply:
removed unused constants set type and descrition